tensorboard/webapp/notification_center/BUILD

```diff
@@ -0,0 +1,36 @@
+load("//tensorboard/defs:defs.bzl", "tf_ng_module", "tf_ts_library")
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])
+
+tf_ng_module(
+    name = "notification_center",
+    srcs = [
+        "_notification_center_module.ts",
+    ],
+    assets = [
+        "//tensorboard/webapp/notification_center/_views:notification_center_styles",
+        "//tensorboard/webapp/notification_center/_views:html",
+    ],
+    deps = [
+        "//tensorboard/webapp/notification_center/_redux",
+        "//tensorboard/webapp/notification_center/_redux:types",
+        "//tensorboard/webapp/notification_center/_views",
+        "@npm//@angular/core",
+        "@npm//@ngrx/effects",
+        "@npm//@ngrx/store",
+    ],
+)
+
+tf_ts_library(
+    name = "types",
+    srcs = [
+        "types.ts",
+    ],
+    deps = [
+        "//tensorboard/webapp:app_state",
+        "@npm//@ngrx/store",
+        "@npm//rxjs",
```

Comment:
I don't think any of these deps are being used. Do a pass and vet only things you need.

Reply:
Omit them. Also do a pass in other `BUILD` files